tests/components/prometheus/helpers.py

```diff
@@ -0,0 +1,170 @@
+"""Helper methods for prometheus tests."""
+
+
+class MetricsTestHelper:
+    """Helps with formatting prometheus metrics and future-proof label changes."""
+
+    @classmethod
+    def get_device_class_label_line(cls, device_class):
+        """Format the device_class label line (tries to account for enum class)."""
+        if device_class:
+            device_class = str(device_class)
+        return f'device_class="{device_class or ""}",'
+
+    @classmethod
+    def _get_metric_string(
+        cls,
+        metric_name,
+        domain,
+        friendly_name,
+        object_id,
+        metric_value=None,
+        area=None,
+        device_class=None,
+    ):
+        device_class_label_line = cls.get_device_class_label_line(device_class)
+        final_metric_value = f" {metric_value}" if metric_value else ""
```

Comment:
Similar to my comment about `positive_comparison`, perhaps assertions without a value should be handled by separate functions.